### ISSUE TITLE
No plugins installed or found by other hosts

### ISSUE THREAD

**Issue by korakios** [flathub/org.surge_synth_team.surge-xt#8]
Hi,
after installing surge-xt flatpak , Carla or Ardour can't find vst3 or lv2 version 
(Both Calra and Ardour are flatpaks and can see Cardinal flatpak plugin.
As standalone it works fine , I have no clue what info to provide , sorry about that
I'm on KDE Neon latest 
`flatpak list | grep surge`
`Surge XT Synthesizer    org.surge_synth_team.surge-xt   1.1.2   stable  system`

**Comment by hfiguiere** [comment]
because this is only the standalone app. plugins are a different package.

**Comment by hfiguiere** [comment]
You are looking for `org.freedesktop.LinuxAudio.Plugins.Surge-XT` which is listed if you do `flatpak search Surge`

**Comment by hfiguiere** [comment]
Also searching Surge in _GNOME Software_:

![image](https://user-images.githubusercontent.com/114441/207433004-9bc9f490-bc6d-447f-a978-85aa27be53b7.png)

And when looking at the Surge-XT package page in _GNOME Software_
![image](https://user-images.githubusercontent.com/114441/207433133-7c27e31d-98ef-48fb-aba7-c25cec5eeae0.png)

**Comment by hfiguiere** [comment]
and if you "grep" the list of packages, at least "grep -i" so that you don't get caught in capitalisation shenanigans.

**Comment by korakios** [comment]
My mistake ,I though for some reason that "Surge XT Synthesizer org.surge_synth_team.surge-xt" was also installing the plugins
But I did install the XT plugin version previously and re-add it , same problem
`Surge XT Synthesizer LV2/VST3/CLAP      org.freedesktop.LinuxAudio.Plugins.Surge-XT     1.1.1   21.08   system`

Is there a different repo? should I post on 
https://github.com/flathub/org.freedesktop.LinuxAudio.Plugins.Surge-XT
instead?
Edit: I just noticed the other repo  "This repository has been archived by the owner before Nov 9, 2022. It is now read-only"
